client/actions/documentActions.js

```diff
@@ -0,0 +1,132 @@
+import axios from 'axios';
+import * as types from './actionTypes';
+
+/**
+ * Load document success action creator
+ * @export
+ * @param {any} document
```

Comment:
Is this going to any or an array? Remember The state is immutable. I can see you made an excessive use of "any", try and correct this.